.github/workflows/upload_all.yml

```diff
@@ -86,12 +97,10 @@ jobs:
         with:
           serviceAccountJsonPlainText: ${{ secrets.POLYPOD_ANDROID_STORE_SERVICE_ACCOUNT_JSON }}
           packageName: coop.polypoly.polypod
-          releaseName: ${{ steps.vars.outputs.release_name }}
+          releaseName: ${{ steps.determine-release-name.outputs.release_name }}
```

Comment:
Technically an unrelated fix, but given how tedious it is to test this kinda stuff, I figured I'd include it here. Sets the release name correctly on Android.